src/canonmn_int.cpp

```diff
@@ -2068,6 +2128,24 @@ namespace Exiv2 {
         return os;
     }
 
+    std::ostream& CanonMakerNote::printSi0x000c(std::ostream& os,
+                                                const Value& value,
+                                                const ExifData*)
+    {
+        if (value.toLong() == 0) return os << "--";
```

Comment:
[style] Please break the **if** statement so that the `return` instruction goes in a separate line (and also use brackets even for **if** statements with 1 instruction.